packages/core/src/support/computerTester/ComputerTester.ts

```diff
@@ -89,8 +89,10 @@ export class ComputerTester {
       output: this.outputDevice,
       params: new Proxy({}, {
         get: (_, key: string) => {
-          const param = this.computer.params.find(p => p.name === key);
+          const params = this.makeParams();
+          const param = params.find(p => p.name === key) as Param;
           if (!param) throw new Error(`Param "${key}" does not exist`);
```

Comment:
The param here should come from the value returned by `this.makeParams`. 
However, the original merge method directly modified the reference of `this.computer.params`, which is why it accidentally worked correctly.